web/dialog.css

```diff
@@ -293,7 +293,8 @@
       align-self: stretch;
 
       > div {
-        &::before, > div {
+        &::before,
+        > div {
```

Comment:
Sorry, but why was this changed since it looks unrelated to the patch?

Reply:
Previously `.messageBar` had more vertical padding, that @ryzokuken had to reduce because of the `Undo` button. This `margin-block` preserves the old spacing in the `.messageBar` usage for the new alt text dialog.

However, the one at line 307 is not needed since it's just duplicating this one.

Reply:
@Snuffleupagus I overlooked that, now the first commit has the reformatting and doesn't fail the linter.